# HG changeset patch
# User Rik <rik@octave.org>
# Date 1397874382 25200
#      Fri Apr 18 19:26:22 2014 -0700
# Node ID 1fc22871bd8b03ddbd947458a4fdf7788c6b8a1f
# Parent  cce0a63afb3c1fddb64674bddb1d972adc45a754
image.m: Set axes layer to "top" for compatability with Matlab.

image.m: Set axes layer to "top" for compatability with Matlab.

diff --git a/scripts/image/image.m b/scripts/image/image.m
--- a/scripts/image/image.m
+++ b/scripts/image/image.m
@@ -193,17 +193,17 @@ function h = __img__ (hax, do_new, x, y,
           mn = intmin (cls);
           mx = intmax (cls);
           set (hax, "clim", double ([mn, mx]));
         endif
       endif
 
     endif  # ! isempty (img)
 
-    set (hax, "view", [0, 90], "ydir", "reverse", "layer", "bottom");
+    set (hax, "view", [0, 90], "ydir", "reverse", "layer", "top");
 
   endif  # do_new
 
 endfunction
 
 
 %!demo
 %! clf;
